crates/miden-lib/src/transaction/mod.rs

```diff
@@ -71,6 +73,12 @@ impl TransactionKernel {
             .expect("failed to deserialize transaction kernel runtime")
     }
 
+    pub fn tx_script_main() -> Program {
+        // TODO: make static
+        Program::read_from_bytes(TX_SCRIPT_MAIN_BYTES)
+            .expect("failed to deserialize tx script executor runtime")
+    }
+
```

Comment:
Maybe this would be a good opportunity to make all of these kernel-related functions use a static, following the approach of the note scripts in `crates/miden-lib/src/note/scripts.rs`.